plutus-ir/src/Language/PlutusIR/Parser.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE FlexibleContexts  #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE Rank2Types        #-}
```

Comment:
I usually enable `RankNTypes` as `Rank2Types` [is a lie](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/glasgow_exts.html#arbitrary-rank-polymorphism).